cupyx/scipy/sparse/coo.py

```diff
@@ -116,8 +116,12 @@ def __init__(self, arg1, shape=None, dtype=None, copy=False):
 
             self.has_canonical_format = False
 
+        elif base.isdense(arg1):
```

Comment:
`base.isdense` does not check `ndim <= 2`.  Could you follow the way in `compressed.py`?
https://github.com/cupy/cupy/blob/39ddc1cc94d25528671f10732f958202010d136b/cupyx/scipy/sparse/compressed.py#L371-L373